formatters/jdt-core/build.properties

```diff
@@ -1,3 +1,20 @@
+#
+# Copyright 2010-2016. All work is copyrighted to their respective
```

Comment:
This is pretty standard and we had it on all java just not everything else.  The purposr is so all files are branded.  Separate licensing in root will exist to.  What is the concern for users?  This is entirely automated so it shouldnt be a real concern.


Get Outlook for Android






On Tue, May 10, 2016 at 10:10 AM -0700, "Christopher Tubbs" <notifications@github.com> wrote:





> @@ -1,3 +1,20 @@
> +#
> +# Copyright 2010-2016. All work is copyrighted to their respective

We shouldn't put the copyright portion in the license headers. We should put those in the NOTICE file reference in the Apache LICENSE. It's friendlier for downstream projects.

---
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub:
https://github.com/revelc/formatter-maven-plugin/pull/105/files/4c1c96ae10d28eaacbd3e77fd3504bfb3f8aefd5#r62712186